ibm/data_source_ibm_is_vpc_default_routing_table.go

```diff
@@ -0,0 +1,188 @@
+package ibm
+
+import (
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+const (
+	isDefaultRoutingTableID             = "default_routing_table"
+	isDefaultRoutingTableHref           = "href"
+	isDefaultRoutingTableName           = "name"
+	isDefaultRoutingTableResourceType   = "resource_type"
+	isDefaultRoutingTableCreatedAt      = "created_at"
+	isDefaultRoutingTableLifecycleState = "lifecycle_state"
+	isDefaultRoutingTableRoutesList     = "routes"
+	isDefaultRoutingTableSubnetsList    = "subnets"
+	isDefaultRTVpcID                    = "vpc"
+	isDefaultRTDirectLinkIngress        = "route_direct_link_ingress"
+	isDefaultRTTransitGatewayIngress    = "route_transit_gateway_ingress"
+	isDefaultRTVPCZoneIngress           = "route_vpc_zone_ingress"
+	isDefaultRTDefault                  = "is_default"
+)
+
+func dataSourceIBMISVPCDefaultRoutingTable() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceIBMISVPCDefaultRoutingTableGet,
+		Schema: map[string]*schema.Schema{
+			isDefaultRTVpcID: {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "VPC identifier",
+			},
+			isDefaultRoutingTableID: {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Default Routing Table ID",
+			},
+			isDefaultRoutingTableHref: {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Default Routing table Href",
+			},
+			isDefaultRoutingTableName: {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Default Routing table Name",
+			},
+			isDefaultRoutingTableResourceType: {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Default Routing table Resource Type",
+			},
+			isDefaultRoutingTableCreatedAt: {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Default Routing table Created At",
+			},
+			isDefaultRoutingTableLifecycleState: {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Default Routing table Lifecycle State",
+			},
+			isDefaultRTDirectLinkIngress: {
+				Type:        schema.TypeBool,
```

Comment:
Remove optional and force new..
It shd be computed

Reply:
simialr for all other attributes

Reply:
Done